packages/template/src/index.js

```diff
@@ -77,7 +77,7 @@ export default {
 		jsep.hooks.add('gobble-token', gobbleTemplateLiteral);
 
 		jsep.hooks.add('after-token', function gobbleTaggedTemplateIdentifier(env) {
-			if (env.node.type === jsep.IDENTIFIER && this.code === BTICK_CODE) {
+			if ((env.node.type === jsep.IDENTIFIER || env.node.type === jsep.MEMBER_EXP) && this.code === BTICK_CODE) {
```

Comment:
I wonder if it should be more like `(env.node.type === jsep.MEMBER_EXP && env.node.object.name === 'String' && env.node.property.name === 'raw')` so that it only works with String.raw? Or throw an error if it's not `String.raw`?

Reply:
@6utt3rfly thanks for looking into my PR that quickly. 

IMO, I would not restrict it to `String.raw` as one can also cluster his custom template functions in objects, e.g.:

```js
    const MyLitFuntions = {
      test: function(strings, ...values) {
        return `${strings.join()} - ${values.join()}`;
      }
    };

    console.log(MyLitFuntions.test`Hello ${s}`);
```

Supporting `MemberExpression`s in general would ensure that the template parser is open enough to support this. One thing I could imagine in addition is to drill down in the `MemberExpression` to check whether the tail is an `Identifier`.

Reply:
Thanks for that example, @petermuessig - I'll approve and merge. Thank you again for your contribution!

Reply:
Thanks @6utt3rfly for your fast support. Now I can get rid of my workaround again... 😄